src/index.html

```diff
@@ -8,9 +8,46 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="form form--big"
+      action="/"
+      method="post"
+      data-qa="big"
+      >
+
+      <div class="search-bar search-bar--big">
+        <input
+          name="big-frame"
+          data-qa="keypress"
+          class="search-bar__input search-bar__input--big"
+          type="text"
+          placeholder="Try “Los Angeles”"
+          >
+      </div>
+    </form>
+
+
+
+    <form
+    class="form form--small"
+    action="/"
+    data-qa="small"
+    method="post"
+    >
```

Comment:
```suggestion
    <form
      class="form form--small"
      action="/"
      data-qa="small"
      method="post"
    >
```

Reply:
Still not fixed from the previous review
<img width="910" alt="image" src="https://github.com/mate-academy/layout_search-bar-airbnb/assets/64750958/b62df173-b79f-4dbe-a0ee-bc16973fc1f4">